src/allmydata/util/eliotutil.py

```diff
@@ -246,53 +214,6 @@ def parse(self, description):
         else:
             return parser(kind, args)
 
-    def _get_arg(self, arg_name, default, arg_list):
-        return dict(
-            arg.split(u"=", 1)
-            for arg
-            in arg_list
-        ).get(
-            arg_name,
-            default,
-        )
-
-    def _parse_file(self, kind, arg_text):
-        # Reserve the possibility of an escape character in the future.  \ is
-        # the standard choice but it's the path separator on Windows which
-        # pretty much ruins it in this context.  Most other symbols already
-        # have some shell-assigned meaning which makes them treacherous to use
-        # in a CLI interface.  Eliminating all such dangerous symbols leaves
-        # approximately @.
-        if u"@" in arg_text:
-            raise ValueError(
-                u"Unsupported escape character (@) in destination text ({!r}).".format(arg_text),
-            )
-        arg_list = arg_text.split(u",")
-        path_name = arg_list.pop(0)
-        if path_name == "-":
-            get_file = lambda: stdout
-        else:
-            path = FilePath(path_name)
-            rotate_length = int(self._get_arg(
-                u"rotate_length",
-                1024 * 1024 * 1024,
-                arg_list,
-            ))
-            max_rotated_files = int(self._get_arg(
-                u"max_rotated_files",
-                10,
-                arg_list,
-            ))
-            def get_file():
-                path.parent().makedirs(ignoreExistingDirectory=True)
-                return LogFile(
-                    path.basename(),
-                    path.dirname(),
-                    rotateLength=rotate_length,
-                    maxRotatedFiles=max_rotated_files,
-                )
-        return lambda reactor: FileDestination(get_file())
-
```

Comment:
This deletion looks wrong to me.  Was it automatically determined by a tool that didn't notice the `getattr(self, u"_parse_{}".format(kind))` above?

Reply:
My fault, I just searched with grep manually.